src/main/java/io/mantisrx/api/tunnel/CrossRegionHandler.java

```diff
@@ -88,7 +88,7 @@
     private Subscription subscription = null;
     private final DynamicIntProperty queueCapacity = new DynamicIntProperty("io.mantisrx.api.push.queueCapacity", 1000);
     private final DynamicIntProperty writeIntervalMillis = new DynamicIntProperty("io.mantisrx.api.push.writeIntervalMillis", 50);
-	private final DynamicStringProperty tunnelRegionsProperty = new DynamicStringProperty("io.mantisrx.api.tunnel.regions", "");
+	private final DynamicStringProperty tunnelRegionsProperty = new DynamicStringProperty("io.mantisrx.api.tunnel.regions", Util.getLocalRegion());
 
 	private List<String> getTunnelRegions() {
 		return Arrays.asList(tunnelRegionsProperty.get().split(","));
```

Comment:
nit: possibly convert this to lowercase as well, maybe also `trim` just in case?